ml/backend/ggml/ggml/src/ggml-cpu/cpu.go

```diff
@@ -1,5 +1,6 @@
 package cpu
 
+// #cgo CFLAGS: -Wno-implicit-function-declaration
```

Comment:
This was causing build errors on windows

Reply:
Can you elaborate? I haven't seen those errors before

Reply:
I only see it on Windows with MinGW. I'll remove and we can add it back in a follow up if its needed